README.rst

```diff
@@ -80,7 +80,7 @@ corresponding image pull tags and their release notes where applicable:
 +--------------------------------------------------------------------------------------+------------+----------------------------------------------+-------------------------------------------------------------------------------------------+
 | `main <https://github.com/cilium/cilium/commits/main>`__                             | daily      | ``quay.io/cilium/cilium-ci:latest``          | N/A                                                                                       |
 +--------------------------------------------------------------------------------------+------------+----------------------------------------------+-------------------------------------------------------------------------------------------+
-| `v1.15.0-rc.1 <https://github.com/cilium/cilium/commits/v1.15.0-rc.1>`__             | 2024-01-16 | ``quay.io/cilium/cilium:v1.15.0-rc.1``       | `Release Candidate Notes <https://github.com/cilium/cilium/releases/tag/v1.15.0-rc.1>`__  |
+| `v1.15.0-pre.3 <https://github.com/cilium/cilium/commits/v1.15.0-pre.3>`__           | 2023-12-04 | ``quay.io/cilium/cilium:v1.15.0-pre.3``      | `Release Candidate Notes <https://github.com/cilium/cilium/releases/tag/v1.15.0-pre.3>`__ |
```

Comment:
Do we really want this change?

Reply:
I'd say let's keep `v1.15.0-rc-1`, and we should look into fixing the script as a follow-up

Reply:
The script fails because it picks the latest tag on the branch, but `v1.15.0-rc-1` is on `v1.15`, not on `main`.